htdocs/core/lib/functions.lib.php

```diff
@@ -10197,6 +10198,74 @@ function natural_search($fields, $value, $mode = 0, $nofirstand = 0)
 	return $res;
 }
 
+/**
+ * Returns simple order table template as string
+ *
+ * @param	Translate	$outputlangs	Output language
+ * @param   Object		$object			Object
+ * @return	string						template
+ */
+function showSimpleOrderTable($outputlangs, $object)
+{
+
+	global $conf;
+
+	$discountIsAvailable = false;
+	
+	foreach($object->lines as $order_position) { 
```

Comment:
What if object has no ->lines property ?

Reply:
@eldy i included a condition if lines property is empty, but why the test is failing? https://app.travis-ci.com/github/Dolibarr/dolibarr/builds/265505646 - can you explain it? Where is Task::$price? :)

Reply:
@eldy any ideas?

Reply:
> @eldy i included a condition if lines property is empty, but why the test is failing? https://app.travis-ci.com/github/Dolibarr/dolibarr/builds/265505646 - can you explain it? Where is Task::$price? :)

This seems introduced by the new function line 10245.
When object is not an Order but Task. The field ->line->price does not exists ...

Reply:
It should be possible in both variants in my thoughts. I dont't use ODT-Files - it is explicit a digital process. Nothing to print and no attachments. It would be a very nice feature to list the order in an email. I have seen that as order confirmation on many online shops.

Reply:
@eldy is it ok for you?

Reply:
@eldy reminder :)